### PR TITLE
Promotions. Everyone gets a medal

### PR DESCRIPTION
**Pull request by Inscrutable**
worm424 => Docs
phase => Ore Back-end Manager

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/a7844ce/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/a7844ce/about/staff.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/726#issuecomment-408583241), not to the direct url to the preview._

File: source/about/staff.rst [hunk 1]
```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
...
 |                                   | Website Manager                    |
 +-----------------------------------+------------------------------------+
+| phase                             | Ore Back-end Manager               |
++-----------------------------------+------------------------------------+
 | phroa                             | IRC Manager                        |
 +-----------------------------------+------------------------------------+
 | progwml6                          | Ore Back-end Manager               |
 |                                   |                                    |
-|                                   | Systems Management                 |
+|                                   | Systems Manager                    |
```
Comment (ST-DDT) [inline source/about/staff.rst]: Maybe `System Manager`, but I'm not sure.

**Review by ST-DDT — APPROVED** [review]

File: source/about/staff.rst (hunk 1 above)
Comment (Inscrutable) [reply source/about/staff.rst]: I'm just mirroring the phrasing from lukegb's position.

File: source/about/staff.rst [hunk 2]
```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
 |                                   |                                    |
 |                                   | Website Manager                    |
 +-----------------------------------+------------------------------------+
+| phase                             | Ore Back-end Manager               |
```
Comment (gabizou) [inline source/about/staff.rst]: s/Ore Back-end Manager/ Ore Development Manager

File: source/about/staff.rst [hunk 3]
```diff
@@ -53,16 +52,17 @@ The **Project Managers** of Sponge each coordinate a separate aspect of the Spon
...
++-----------------------------------+------------------------------------+
 | phroa                             | IRC Manager                        |
 +-----------------------------------+------------------------------------+
 | progwml6                          | Ore Back-end Manager               |
```
Comment (gabizou) [inline source/about/staff.rst]: nix the Ore Back-end Manager bit, since it's bundled with Systems Manager.

File: source/about/staff.rst (hunk 2 above)
Comment (gabizou) [reply source/about/staff.rst]: And mbaxter = Ore Community Manager, right @Zidane ?

Comment (phase) [reply source/about/staff.rst]: `Ore Development Manager`

File: source/about/staff.rst (hunk 3 above)
Comment (phase) [reply source/about/staff.rst]: `Ore Review Team Manager`

File: source/about/staff.rst (hunk 2 above)
Comment (phase) [reply source/about/staff.rst]: `Ore Development Manager`

**Review by phase — CHANGES_REQUESTED** [review]
these titles are clear

**Comment by phase** [comment]
oh wait I commented on the wrong person! mbax shall be the `Ore Review Team Manager`

**Review by ST-DDT — APPROVED** [review]

**Comment by phase** [comment]
Do we want to add the `Sysadmin` title to Snowie & kashike?

**Comment by Zidane** [comment]
@phase Yes.

**Comment by Inscrutable** [comment]
Is `Sysadmin` a new role, and if so, where does it fit (a new section below *project managers*)?
We already have two `Systems Manager`s. I'm starting to wonder what all these things really mean.

**Comment by ST-DDT** [comment]
A list of positions and their purpose would really be nice.

**Review by phase — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by mbax — COMMENTED** [review]
Great! I'm late, but I wanted to feel like I was a part of it!

**Comment by Inscrutable** [comment]
Well, you *did* give me the list of Ore Reviewers to publish, so I figured that was approval.

**Pull request merged**